Miraheze/Mirahezelocalset.php

```diff
@@ -404,6 +404,16 @@
 			),
 			'superadmin' => array(
 				'admin',
+	),		),
```

Comment:
Please move these two ), to separate lines and align them with the arrays they close